test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp/Examples/ParameterModelExample.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Collections.Generic;
+
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Abstractions;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Resolvers;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp.Models;
+using Newtonsoft.Json.Serialization;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp.Examples
+{
+    class ParameterModelExample : OpenApiExample<string>
```

Comment:
It should be `StringParameterExample` because it only takes care of `<string>`.

Then, you should create other types such as `Int32ParameterExample : OpenApiExample<int>`, `BooleanParameterExample : OpenApiExample<bool>`, ...

Here's an example:

```csharp
public class DateTimeParameterExample : OpenApiExample<DateTime>
{
    public override IOpenApiExample<DateTime> Build(NamingStrategy namingStrategy = null)
    {
        this.Examples.Add(OpenApiExampleResolver.Resolve("dateTimeValue1", DateTime.Parse("2021-01-01"), namingStrategy));
        this.Examples.Add(OpenApiExampleResolver.Resolve("dateTimeValue2", DateTime.Parse("2021-01-01T12:34:56Z"), namingStrategy));

        return this;
    }
}
```